.github/pull_request_template.md

```diff
@@ -6,8 +9,7 @@
   - [ ] Run the production version of Accessibility Insights for Windows against a version with changes.
   - [ ] Attach any screenshots / GIF's that are applicable.
 
-Note - After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge. 
+> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge. 
```

Comment:
Why the introduction of '`>`'?

Reply:
It gives more emphasis for the note thing as a `PS` (Post Script). I mean thats the actual use of `>` in markdown!